nautobot_golden_config/signals.py

```diff
@@ -1,11 +1,37 @@
 """Signal helpers."""
-
+from django.apps import apps as global_apps
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from nautobot.dcim.models import Platform
 from nautobot_golden_config import models
 
 
+def post_migrate_create_statuses(sender, apps=global_apps, **kwargs):  # pylint: disable=unused-argument
+    """Callback function for post_migrate() -- create Status records."""
+    Status = apps.get_model("extras", "Status")  # pylint: disable=invalid-name
+    ContentType = apps.get_model("contenttypes", "ContentType")  # pylint: disable=invalid-name
+    for status_config in [
+        {
+            "name": "Accepted",
+            "slug": "accepted",
+            "defaults": {
+                "description": "Config plan is accepted",
+                "color": "4caf50",  # Green
+            },
+        },
+        {
+            "name": "Not Accepted",
```

Comment:
Don't we need three statuses:
1) Unapproved (pre-decision)
-> 2a) Approved
-> 2b) Decliend/denied etc. (decision to drop, ie long manually created plan would have to be fixed by the engineer instead of deleted and recreated )

Reply:
> ie long manually created plan would have to be fixed by the engineer instead of deleted and recreated

The config set is not currently able to be modified via UI, so the only solution would be to delete and recreate it. I didn't see anything in the user stories about needing to be able to modify the config set once it has been generated.

Reply:
I prefer `Not Approved` over `Not Accepted` and `Unapproved`, I realize Not approved and Unapproved are the same, but having `Not` in it seems more explicit to me.